client/components/editor.vue

```diff
@@ -121,12 +131,26 @@
         fixToolbarToTop: false,
         originalToolbarYPos: -1,
         toolbarWidth: {width: '100%'},
+        cursorsHandle: null,
+        dipatch: null,
+        state: null,
       };
     },
-
+    watch: {
+      focusedCursor(cursor) {
```

Comment:
List all arguments to a function, even if you do not need them. For example, here there should be `newCursor`, `oldCursor`.